bin/delete_pr_comment

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash -eu
+
+# This script is used to delete comments on a pull request, and must be run in a buldkite PR step.
+#
+# Example:
+#   delete_pr_comment --github-token <token> --containing "part of a comment"
+
+# Check dependencies and print their versions for diagnosis purposes
+jq --version
+curl --version
```

Comment:
How do you feel about adding an empty line here so that the version diagnostics info are separated from the rest of the script output?

```suggestion
curl --version
# Print line to separate versions info from the rest of the script output
echo ''
```

Reply:
Updated in https://github.com/Automattic/a8c-ci-toolkit-buildkite-plugin/pull/52/commits/5ad7153d746b7a0a20287cafc284707e2784bf89